io/private/las/Summary.cpp

```diff
@@ -57,7 +57,8 @@ void Summary::addPoint(double x, double y, double z, int returnNumber)
     m_bounds.grow(x, y, z);
 
     // Returns numbers are indexed from one, but the array indexes from 0.
-    m_returnCounts[returnNumber - 1]++;
+    if (returnNumber >= 1 && returnNumber <= m_returnCounts.size())
+        m_returnCounts[returnNumber - 1]++;
```

Comment:
returnNumber of 0 happens in files even though it is invalid and is not supposed to happen. I understand we don't want to access the array OB, but we do want histogram counts of `returnNumber == 0`.